recipes/qt-multimedia/meta.yaml

```diff
@@ -0,0 +1,113 @@
+  missing_dso_whitelist:  # [win]
+    - '$RPATH/api-ms-win-core-libraryloader-l1-2-0.dll'  # [win]
+    - '$RPATH/api-ms-win-ntuser-sysparams-l1-1-0.dll'  # [win]
+    - '$RPATH/api-ms-win-core-heap-l2-1-0.dll'  # [win]
+    - '$RPATH/api-ms-win-core-libraryloader-l1-2-1.dll'  # [win]
+
+requirements:
+  build:
+    - make                               # [unix]
+    - cmake
+    - ninja
+    - perl
+    - ruby >=2.5                         # [linux]
+    - bison                              # [linux]
+    - flex                               # [linux]
+    - gperf                              # [linux]
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - qt6-main {{ version }}             # [build_platform != target_platform]
+    - {{ cdt('mesa-libgl-devel') }}      # [linux]
+    - {{ cdt('mesa-libegl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}      # [linux]
+    - {{ cdt('libdrm-devel') }}          # [linux]
+    - {{ cdt('libglvnd-glx') }}          # [linux]
+    - {{ cdt('libglvnd-egl') }}          # [linux]
+    - {{ cdt('libglvnd-devel') }}        # [linux]
+    # native libs
+    - ffmpeg                             # [build_platform != target_platform]
+    - gst-plugins-base {{ gstreamer }}   # [build_platform != target_platform and linux]
+    - gstreamer                          # [build_platform != target_platform and linux]
+    - pulseaudio                         # [(build_platform != target_platform) and linux]
+
+    - xorg-libx11                       # [(build_platform != target_platform) and linux]
+    - xorg-libxext                      # [(build_platform != target_platform) and linux]
+    - xorg-libxdamage                   # [(build_platform != target_platform) and linux]
+    - xorg-libxrandr                    # [(build_platform != target_platform) and linux]
+    - xorg-libxcomposite                # [(build_platform != target_platform) and linux]
+    - xorg-libxcursor                   # [(build_platform != target_platform) and linux]
+    - xorg-libsm                        # [(build_platform != target_platform) and linux]
+    - xorg-libice                       # [(build_platform != target_platform) and linux]
+    - xorg-libxxf86vm                   # [(build_platform != target_platform) and linux]
+    - xorg-libxtst                      # [(build_platform != target_platform) and linux]
+    - xorg-xf86vidmodeproto             # [(build_platform != target_platform) and linux]
+  host:
+    - libvulkan-headers                  # [not osx]
+    - libvulkan-loader                   # [not osx]
+    - pthread-stubs                      # [linux]
+    - gst-plugins-base  {{ gstreamer }}  # [unix]
+    # gstreamer plugin not supported on win32: https://bugreports.qt.io/browse/QTBUG-107073
+    - gstreamer                          # [unix]
+    - ffmpeg
+    - pulseaudio                         # [linux]
+    - alsa-lib                           # [linux]
```

Comment:
Is there some discernible order of precedence/priority here? If so, perhaps leave a comment? If not, can we sort them alphabetically?